File: src/main/java/com/rukka/Hello.java
package com.rukka;

public class Hello {
    public static void main(String[] args) {
//        System.out.println("Hello Java");
        PersonJava p = new PersonJava(66.5f, 1.7f);
        p.hello();
//        p.weight = 66.5f;
//        p.height = 1.7f;
        System.out.println(p.bmi());
        System.out.println(("abcde").charAt(4));
        System.out.println(("abcde").substring(1, 4) + " " + ("1234567").length());
        /*int age = 19;
        Integer age2 = 19;
        char c = 'A';
        char c = 'A';
        Character c2 = 'B';
        byte b = 120;
        float weight = 66.5f;
//        Float
        double height = 1.7;
//        Double
        boolean enrolled = true;
//        Boolean
        String name = "Tom";*/
    }
}

File: src/main/kotlin/com/rukka/Hello.kt
package com.rukka

class PersonKotlin(var name: String = "", var weight: Float, var height: Float) {
    init {
        println()
    }
//    constructor(name: String, weight: Float, height: Float): this(weight, height)
    fun bmi() = weight/(height*height)
    fun hello() {
        println("Hello Kotlin")
    }
}

fun main() {
//    println("Hello Kotlin")
    val pK = PersonKotlin(weight = 66.5f, height = 1.7f)
//    pK.hello()
    println(pK.bmi())
    val c = 'A'
    println(c.toInt())

    var s: String? = null
    print("${s?.length} ${s?.get(2)} ${s?.substring(1, 5)}")


    /*var age = 19
    age = 20
    var weight = 66.5f
    var name: String
    name = "Rukka"*/
}